packages/babel-plugin-open-source/babel.js

```diff
@@ -32,7 +45,7 @@ module.exports = declare(api => {
 
       const location = path.container.openingElement.loc
       let url = null;
-      let editor = state.opts?.editor?.toLowerCase() || 'vscode';
+      let editor = (get(state, 'opts.editor') || 'vscode').toLowerCase();
```

Comment:
can we keep it simple and use if `(state.opts && state.opts.editor)`?

Reply:
got it, sorry for late reply.